clients/githubrepo/stats/stats.go

```diff
@@ -15,29 +15,50 @@
 package stats
 
 import (
-	"go.opencensus.io/stats"
-	"go.opencensus.io/stats/view"
-	"go.opencensus.io/tag"
+	"sync"
+
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/metric"
 )
 
+var Meter = otel.Meter("ossf-scorecard")
+
+var Metrics struct {
+	RemainingTokens metric.Int64Histogram
+	RetryAfter      metric.Int64Histogram
+}
+
+var once sync.Once
+
+func InitMetrics() error {
+	var err error
+	once.Do(func() {
+		c, err := Meter.Int64Histogram("RetryAfter",
+			metric.WithDescription("Measures the retry delay when dealing with secondary rate limits"),
+			metric.WithUnit("s"))
+		if err != nil {
+			return
```

Comment:
I think there's some unintentional shadowing here since you use `c, err :=` above